unison-src/transcripts/fix2268.md

```diff
@@ -13,7 +13,7 @@ unique ability A where
 unique ability B where
   a : Char
 
-test : 'Nat
+test : () -> Nat
```

Comment:
Why this change?

Reply:
`'Nat` is now equivalent to `'{} Nat`, but I believe this example relies on inference of abilities on that arrow.